### ISSUE TITLE
runtime: detect zombie goroutines

### ISSUE THREAD

**Issue by dvyukov** [golang/go#5308]
<pre>Runtime can detect goroutines blocked on unreachable chans/mutexes/etc and report such
cases.
This needs an interface in http/pprof to query zombie goroutines on running services.
And probably testing package integration to test for absence of zombie goroutines.</pre>

**Comment by ALTree** [comment]
Closing this in favour of the recent #74609 proposal.
